README.md

```diff
@@ -12,9 +11,42 @@ curl -sSL https://raw.githubusercontent.com/cyberark/summon-aws-secrets/master/i
 
 Otherwise, download the [latest release](https://github.com/cyberark/summon-aws-secrets/releases) and extract it to the directory `/usr/local/lib/summon`.
 
-## Usage in isolation
+## Variable IDs
+Variable IDs are used as identifiers for fetching Secrets. These are made up of a secret name (required) and secret key path (optional). 
+
+The format used is `my/secret/name#mysecretkeypath`, where `#mysecretkeypath` is optional and only relevant when the value returned by `my/secret/name` alone is valid JSON.
+
+### secret name (required)
+This is the AWS secret name, which must be ASCII letters, digits, or any of the following characters: /_+=.@-
+
+### secret key path (optional)
+The secret key path to access arbitrarily nested values inside secrets stored as valid JSON.
```

Comment:
I think the `arbitrarily nested values` bit isn't accurate since this only allows you to access top-level keys.